ProjectSettings/Packages/com.unity.learn.iet-framework/Settings.json

```diff
@@ -1,12 +1,15 @@
 {
-    "m_Name": "Settings",
-    "m_Path": "ProjectSettings/Packages/com.unity.learn.iet-framework/Settings.json",
     "m_Dictionary": {
         "m_DictionaryValues": [
             {
                 "type": "System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089",
                 "key": "IET.ShowTutorialsClosedDialog",
                 "value": "{\"m_Value\":false}"
+            },
+            {
+                "type": "System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089",
+                "key": "IET.DisplayWelcomeDialogOnStartup",
+                "value": "{\"m_Value\":false}"
```

Comment:
mmm that seems wrong. After this, are we still getting that welcome dialog?

Reply:
it does seem wrong. I tested setting it to true, but I did not notice any difference. In both cases, the inspector opens Readme.asset on startup, but I don't see any dialog pop up. And the editor automatically sets it to false